webpack.config.js

```diff
@@ -21,7 +21,12 @@ const baseConf = {
     rules: [
       {
         test: /\.jsx?$/,
-        use: ['babel-loader']
+        use: [{
+          loader: 'babel-loader',
+          options: {
+            presets: ['es2015']
```

Comment:
if we're moving this here instead of in `.babelrc` put all the config that's in the babelrc https://github.com/src-d/landing/blob/master/.babelrc